Examples.xcodeproj/xcshareddata/xcschemes/Examples.xcscheme

```diff
@@ -86,6 +86,13 @@
             ReferencedContainer = "container:Examples.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <EnvironmentVariables>
```

Comment:
Please remove this — it obscures potentially important messages from console logs.